roles/openshift_master/defaults/main.yml

```diff
@@ -147,3 +147,6 @@ openshift_master_csr_namespace: openshift-infra
 
 openshift_master_config_file: "{{ openshift_master_config_dir }}/master-config.yaml"
 openshift_master_scheduler_conf: "{{ openshift_master_config_dir }}/scheduler.json"
+
+openshift_master_systemd_enable: yes
+openshift_master_fire_handlers: True
```

Comment:
why is this needed?

Reply:
These are needed in order for us to lay down the systemd files without actually starting the service and executing.  I default the `fire_handlers` to `true` in the normal case but when we are coming through we don't want to have services start up.

Reply:
Ah, I finally understand - there's some route where these implicitly default to false.  I'd be concerned given their names that someone will soon break this for us.  Would something like openshift_master_building_golden_image be more overt about what this is for?

Reply:
Without this the systemd unit files call handlers which start the services.  This is undesirable in our setup.  We just want the units without running the services.

Reply:
agree, my question is now about the name of the 2 vars - wondering if they should be a single one called something like openshift_master_building_golden_image